.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: python
+
+matrix:
+  include:
+     - python: 2.7
+     - python: 3.6
```

Comment:
Maybe test against 3.7? As it's specified in the setup.py as supported.

If 3.7 _and_ 2.7 works, all in between should as well. If we have 3.6, probably 3.7 breaks (new keywords etc.).

Or just add all supported Python versions?

Reply:
Very good point, and trivial to solve.  I've added python 3.7 to the matrix, and swapped it to be the version we deploy from.

Reply:
Wait, no, I take back the comment about it being trivial to add, seems you need to give the CI pipeline sudo permissions or take a completely different approach (https://github.com/travis-ci/travis-ci/issues/9815, https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905).  Are we ok with me reverting the 3.7 change and leaving this just testing 3.6 for now?

Reply:
True, let's keep it simple and just use 3.6.

Reply:
Yeah, that's not trivial and one of the reasons why I set up Azure pipelines for the particle package. Works nicely and tests the code with Py3.7 and most used platforms :-).